packages/base/src/mainview.tsx

```diff
@@ -572,6 +653,25 @@ export class MainView extends React.Component<IProps, IStates> {
       this._model.syncSelectedObject(names, this.state.id);
     }
   }
+
+  private _onKeyDown(event: KeyboardEvent) {
+    // TODO Make these Lumino commands? Or not?
+    if (this._clipSettings.enabled) {
+      switch (event.key) {
+        case 'r':
```

Comment:
Could you update the documentation about the clipping feature and the hotkey?

Reply:
👍🏽 Shall we do this in a separate PR?